src/content/reference/react/useDeferredValue.md

```diff
@@ -501,25 +501,25 @@ input { margin: 10px; }
 
 <DeepDive>
 
-#### How does deferring a value work under the hood? {/*how-does-deferring-a-value-work-under-the-hood*/}
+#### Bagaimana cara menangguhkan nilai bekerja di bawah terpal? {/*how-does-deferring-a-value-work-under-the-hood*/}
```

Comment:
Kemungkinan membuat bingung pembaca, mungkin sugesti ini cocok: 

* `Bagaimana menangguhkan nilai bekerja dibalik layar`

cc @mhaidarhanif @adamcanray

Reply:
Thanks for the suggestion mas @r17x,

I'll use it. the changes is delivered on this commit https://github.com/reactjs/id.react.dev/pull/360/commits/88e7a3b14d901e98e62d188d43dc0ef7b2652064